pkg/test/framework/components/istio/eastwest.go

```diff
@@ -58,15 +59,22 @@ func (i *operatorComponent) deployEastWestGateway(cluster resource.Cluster) erro
 	cmd := exec.Command(genGatewayScript, generateSettings...)
 	cmd.Env = os.Environ()
 	cmd.Env = append(cmd.Env,
+		"ISTIOCTL="+filepath.Join(env.LocalOut, "istioctl"),
```

Comment:
A user should probably be setting PATH to select an istioctl version, but this is for tf compat.